pages/CreatingAnExploration.html

```diff
@@ -31,23 +30,14 @@
       the essence of your exploration.
```

Comment:
Maybe explain how to edit the title? (By clicking "Untitled Exploration" perhaps)